spi/spi_master_slave/README.adoc

```diff
@@ -0,0 +1,140 @@
+= Communicating as master and slave via SPI
+
+This example code shows how to interface two RP2040 microcontrollers to each other using SPI.
+
+== Wiring information
+
+[frame="topbot",options="header"]
+|===
+| Function | Master (RP2040) | Slave (RP2040) | Master (Pico) | Slave (Pico)
+| MOSI | DO0  | DI0  | 25 | 21
+| SCLK | SCK0 | SCK0 | 24 | 24
+| GND  | GND  | GND  | 23 | 23
+| CS   | CS0  | CS0  | 22 | 22
+| MISO | DI0  | DO0  | 21 | 25
+|===
+
+[[spi_master_slave_wiring]]
+[pdfwidth=75%]
+.Wiring Diagram for SPI Master and Slave.
+image::spi_master_slave_bb.png[]
+
+At least one of the boards should be powered, and will share power to the other.
+
+If the master is not connected properly to a slave, the master will report reading all zeroes.
+
+If the slave is not connected properly to a master, it will initialize but never transmit nor receive, because it's waiting for clock signal from the master.
+
+== Outputs
+
+Both master and slave boards will give output to their UART or USB CDC serial port.
```

Comment:
IIRC only UART-stdio is enabled by default, to get USB-stdio as well you need to add something to `CMakeLists.txt`. See chapter 4 of https://datasheets.raspberrypi.org/pico/getting-started-with-pico.pdf

Reply:
I've removed the reference to UART/USB and summarized as stdio.

Resolving.